### PR TITLE
Improve conciseness of README.md

### PR DESCRIPTION
**Pull request by LYCrachel**
### Summary

- "Rails is a web-application framework that includes" -> "Rails as a web-application framework includes" (make clear of the subject)
- "application into three layers, each with a specific responsibility." -> "application into three layers: Model, View, and Controller, each with a specific responsibility." (mention the aspects of the writing in introduction part to give readers a general idea first)
- "is responsible for handling" -> "handles" (be more concise)
- "You can read more" -> "Read more"
- Removed some second-person voice to improve formality

### Other Information

This is from a computer science student trying to improve the writing of the documentation. Thanks :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,49 +1,48 @@
 # Welcome to Rails
 
-Rails is a web-application framework that includes everything needed to
-create database-backed web applications according to the
+Rails as a web-application framework includes everything needed to
```
Comment (utilum) [inline README.md]: Opening with a long pronominal clause is harder to read. Current version looks better to me here.

File: README.md [hunk 2]
```diff
@@ -1,49 +1,48 @@
...
-Ruby classes, or Ruby classes that implement a set of interfaces as provided by
-the Active Model module. You can read more about Active Model in its [README](activemodel/README.rdoc).
+methods. Read more about Active Record in its [README](activerecord/README.rdoc).
+Although most Rails models are backed by a database, models can be ordinary
+Ruby classes or Ruby classes with a set of interfaces provided by
+the Active Model module. Read more about Active Model in its [README](activemodel/README.rdoc).
 
-The _Controller layer_ is responsible for handling incoming HTTP requests and
-providing a suitable response. Usually this means returning HTML, but Rails controllers
+The _Controller layer_ handles incoming HTTP requests and
```
Comment (utilum) [inline README.md]: Current version correctly emphasises the contract, other components can be made to step in and handle, but to keep things clean we _want_ to keep this responsibility in the controller layer.

File: README.md [hunk 3]
```diff
@@ -1,49 +1,48 @@
...
+provides suitable responses. Usually this means returning HTML, but Rails controllers
```
Comment (utilum) [inline README.md]: As above.

File: README.md [hunk 4]
```diff
@@ -1,49 +1,48 @@
...
+are bundled together in Action Pack. Read more about Action Pack in its
 [README](actionpack/README.rdoc).
 
-The _View layer_ is composed of "templates" that are responsible for providing
-appropriate representations of your application's resources. Templates can
-come in a variety of formats, but most view templates are HTML with embedded
-Ruby code (ERB files). Views are typically rendered to generate a controller response,
-or to generate the body of an email. In Rails, View generation is handled by Action View.
-You can read more about Action View in its [README](actionview/README.rdoc).
+The _View layer_ is composed of "templates" that can privide
```
Comment (utilum) [inline README.md]: Responsibility is key here.

Comment (utilum) [reply README.md]: Um... and _typcially_ they are _rendered_. Will you put that back in? 
It's a fine balance, being concise while keeping an eye on explaining MVC.

**Review by utilum — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (LYCrachel) [reply README.md]: I see. Thanks

**Comment by LYCrachel** [comment]
Hi, I have updated README.md according to your suggestion. Thank you if you could accept it ;)

**Comment by utilum** [comment]
I still think the responsibility concept is key, and should be kept, so I would not replace 'is responsible for handling' with 'handles'.
That said, I'm not a committer, and can neither accept nor reject a PR, only review and suggest.

**Comment by LYCrachel** [comment]
Yes I think so, but I've added "is responsible for handling .." back

**Comment by utilum** [comment]
Looks fine to me.

**Comment by fxn** [comment]
Catching up, ACK!

**Comment by fxn** [comment]
I have taken this patch, a revision by @composerinteralia, and considered everything through my personal criteria. The result is bba832d48a8bb3c4e262e707b18e3c7fadb5cf55, which credits you both.

For example, while in our documentation guidelines we [expressely discourage overusing "you"](http://guides.rubyonrails.org/api_documentation_guidelines.html#wording), the sentence

> Read more about Active Record in its README.

sounds a tad imperative to me, while the current

> You can read more about Active Record in its README.

sounds more gentle, it is an invitation. I personally prefer the tone of this one.

In programming jargon a class is not just _with_ and interface, _implements_ is the idiomatic word for that.

And so on.

There is always a tension about conciseness and being too terse. You have to find a sweet spot according to your best judgement, and that is my take.

Thanks for the patch!

**Pull request closed without merging**